### PR TITLE
@orta => Add breadcrumbs to crash reports, especially around ARTiledImageView.

### LINKED ISSUE #408: Add breadcrumb logging to crash reports around - [ARTiledImageView drawRect:]
We need to figure out some things to log so we can try to hone in on how to reproduce these segfaults:

* https://rink.hockeyapp.net/manage/apps/37029/app_versions/33/crash_reasons/33140567
* https://rink.hockeyapp.net/manage/apps/37029/app_versions/33/crash_reasons/32348962
* https://rink.hockeyapp.net/manage/apps/37029/app_versions/33/crash_reasons/32627729

I’m thinking:

* Log (time) when showing a view controller that displays a `ARTiledImageView`
* Log (time) when leaving a view controller that displays a `ARTiledImageView`
* Log (time) when drawing a `ARTiledImageView`
* Log the exact image being displayed

Any other suggestions?

----

Attaching logs should be done like so: http://support.hockeyapp.net/kb/client-integration-ios-mac-os-x/how-to-add-application-specific-log-data-on-ios

### PR DESCRIPTION
**Pull request by alloy**
Fixes #408.

This currently uses unreleased versions of ARAnalytics and HockeyApp, I’ll update it when those are updated and released.

### CODE REVIEW

**Comment by alloy** [comment]
* https://github.com/bitstadium/HockeySDK-iOS/pull/153
* https://github.com/orta/ARAnalytics/pull/155

File: Podfile [hunk 1]
```diff
@@ -87,11 +87,11 @@ pod 'TSMiniWebBrowser@dblock', :head
 pod 'FODFormKit', :git => 'https://github.com/1aurabrown/FODFormKit.git'
 
 # Analytics
-pod 'HockeySDK', '3.5.0'
+pod 'HockeySDK', :git => 'https://github.com/alloy/HockeySDK-iOS.git', :branch => 'develop'
 pod 'Mixpanel', '2.3.1'
-pod 'ARAnalytics/Mixpanel'
-pod 'ARAnalytics/HockeyApp'
-pod 'ARAnalytics/DSL'
+pod 'ARAnalytics/Mixpanel', :git => 'https://github.com/alloy/ARAnalytics.git', :branch => 'add-breadcrumb-logging-to-HockeyApp'
```
Comment (orta) [inline Podfile]: we could switch to the new subspec as an array DSL too

**Comment by eytanbiala** [comment]
@alloy Did you guys find the breadcrumbs helpful in tracking down the crash? I have a few crashes originating at `[UIApplication sendEvent]` that don't include any of our app's code in the stack trace, so I'm looking for a good way to go about investigating those.

**Comment by orta** [comment]
@eytanbiala yeah, it gives you a run through of the events that happened on the way towards that event, https://github.com/artsy/eigen/issues/623

**Comment by alloy** [comment]
@eytanbiala It’s not a guarantee, but the events leading up to crash _may_ definitely help pinpoint a reduction of the crash, like it did in the ticket that @orta pointed to. Obviously it hinges on having enough events being logged or adding your own `ARLog` calls.

**Comment by eytanbiala** [comment]
Thanks! I've swizzled viewWillAppear to add logging to all our v.c.'s, so I at least have a better idea what view the user is on when it happens.

**Comment by alloy** [comment]
@eytanbiala np, hope it helps!

**Pull request merged**